### ISSUE TITLE
17665 - Install and configure Laravel with NGINX - CentOS

### ISSUE THREAD

**Issue by gnakan** [gnakan/gd-cloud-docs#125]

**Comment by susankayjordan** [comment]
Validated

**Comment by kharise** [comment]
For testing, my notes:

I need more direction in the configuration part.  I tried to figure it out on my own but am not able to complete...

For CentOS 7 - here are the 2 parts having to do w/config (and what I did):
#1.
For the configuration in /etc/nginx/nginx.conf…
I added those 2 lines to this location 
(at the end of this example):

http {
    log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
                      '$status $body_bytes_sent "$http_referer" '
                      '"$http_user_agent" "$http_x_forwarded_for"';
….
    # Load modular configuration files from the /etc/nginx/conf.d directory.
    # See http://nginx.org/en/docs/ngx_core_module.html#include
    # for more information.
    index index.php index.html index.htm;
    include /etc/nginx/conf.d/*.conf;

#2.
Step 4 in “Configure Nginx”:
Modify the  php configuration section of the file to look like this: 
location ~ \.php$ {

What if there’s no location ~ \.php$ { section? 
I went to /etc/nginx/default.d/default.conf 
and updated the code in there.  Is that the right thing to do?

Halp!

**Comment by brian-r-miller** [comment]
Yes.  What makes this generic is that I am treating the Nginx configuration file as what it would look like after the includes flesh it out.  Since different distros have different locations for their includes... this makes for some added complexity.  But you seem to have sorted it out.

**Comment by brian-r-miller** [comment]
Well done!   Re-wrote the article to be a little clearer.

**Comment by kharise** [comment]
This was forked from testing - rewritten to make Laravel its own server block (like 00069).
Laravel installed fine, nginx and php are up and running, but I get the a darn 404 for the last step.

few notes:
- This article has the user install PHP v5.5 
- I also tried uncommenting the 2 -mcrypt extensions in my php.ini file (/etc/php.ini)-- to no avail 

@brian-r-miller: 
Per our convo this morning. I'm sure you can make this work... Help!

**Comment by kharise** [comment]
It passed!!!!
